### DISCUSSION TITLE
Embed Prisma Studio inside a Web Application

### DISCUSSION THREAD

**Discussion by VinayaSathyanarayana** [prisma/prisma#5677] [Q&A]
I like Prisma Studio.

- I am looking for documentation/ instructions on how I can embed Prisma Studio in my Web Application Front End for End Users and not in the Admin UI.

- Does Prisma Studio provide access control based on the logged in User?

**Comment by sdnts** [comment]
Hello! This is an important use case in our eyes as well. In fact, we're building this right now! It is probably best to follow this issue for updates: https://github.com/prisma/studio/issues/631

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
